local_settings.py.template

```diff
@@ -0,0 +1,70 @@
+DEBUG = True
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.contrib.gis.db.backends.postgis',
+        'NAME': 'linkedevents',
+        'USER': 'linkedevents',
+        'PASSWORD': 'linkedevents',
+        'HOST': 'linkedevents-db'
+    }
+}
+
+LANGUAGES = (
```

Comment:
I wouldn't have a template with different languages than the repo default, I guess this means there will be extra languages in linkedevents migrations that are not used by the code? Don't know if extra unused fields in the db will cause any trouble, though.

Reply:
I could copypaste settings from other languages, maybe it would be ok for now. :D